features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2223,4 +2223,14 @@
         <MaxAge>{{cors.max_age}}</MaxAge>
         <TagRequests>{{cors.tag_requests}}</TagRequests>
     </CORS>
+
+    <LegacyFeatures>
```

Comment:
dont't we need to add the default set of features that we decided to deprecate to default.json?

Reply:
Fixed 76ad8e0

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3027/commits/76ad8e0307b509aefc1c7e7716756ad143850105